_docs/generators/generic-freemarker-generator.md

```diff
@@ -464,34 +457,33 @@ representing the type:
 ```
 
 ## User Guide
-To use the generic generator based on [Freemarker](https://freemarker.apache.org/) you need two files within your Eclipse workspace:
+To use our generic, [Freemarker](https://freemarker.apache.org/ template-based generator, you need two files within your Eclipse workspace:
 
  * The input CML model (*.cml)
  * The [Freemarker](https://freemarker.apache.org/) template (*.ftl)
  
- With a right-click on the CML file or within the CML editor you can start the generator. You find it in the _Context Mapper_ context menu:
+ With a right-click on the CML file or within the CML editor, you can start the generator. You find it in the _Context Mapper_ context menu:
  
  <a href="/img/generic-generator-context-menu.png">![Generic Textual Generator Context Menu](/img/generic-generator-context-menu.png)</a>
  
- A dialog allows you to select a [Freemarker](https://freemarker.apache.org/) template (*.ftl file) which must be located somewhere in your workspace:
+ A dialog allows you to select a [Freemarker](https://freemarker.apache.org/) template (*.ftl file), which must be located somewhere in your workspace:
  
  <a href="/img/generic-generator-dialog.png">![Generic Textual Generator Dialog](/img/generic-generator-dialog.png)</a>
  
- By finishing the dialog you generate the required file:
+You generate the required file when finishing the dialog:
  
  <a href="/img/generic-generator-result.png">![Generic Textual Generator Dialog](/img/generic-generator-result.png)</a>
 
 ### Freemarker Version
-Context Mapper uses [Freemarker](https://freemarker.apache.org/) in version 2.3.30.
+Context Mapper uses [Freemarker](https://freemarker.apache.org/), currently Version 2.3.30.
 
 ## Example Templates
-With the Context Mapper Eclipse plugin we also deliver example Freemarker templates. Use the _Freemarker Generator Template Examples_ wizard via _File -> New -> Other..._
-to create the sample project containing the Freemarker templates:
+The Context Mapper Eclipse plugin comes with sample Freemarker templates. Use the _Freemarker Generator Template Examples_ wizard via _File -> New -> Other..._ to create the sample project containing the Freemarker templates: <!-- TODO is this still current? -->
 
 <a href="/img/screenshot-new-freemarker-example-project-1.png">![Create Sample Project with Freemarker Templates (1)](/img/screenshot-new-freemarker-example-project-1.png)</a>
 
 <a href="/img/screenshot-new-freemarker-example-project-2.png">![Create Sample Project with Freemarker Templates (2)](/img/screenshot-new-freemarker-example-project-2.png)</a>
 
 The project currently contains the following example templates:
 
- * Ubiquitous language glossary written in Markdown
+ * Ubiquitous language glossary written in Markdown <!-- not really a glossary, more of a full report -->
```

Comment:
@socadk, okay, we have to change this text here or adjust the glossary before merging this PR. (remove comment)

Reply:
TODO Stefan: change wording here